commons/src/main/scala/dagr/commons/io/Io.scala

```diff
@@ -26,19 +26,27 @@ package dagr.commons.io
 import java.io._
 import java.nio.file.{Files, Path}
 
+import scala.collection.mutable.ListBuffer
+import scala.io.Source
+
+/**
+* Singleton object to provide access to Io utility methods.
+*/
+object Io extends IoUtil
+
 /**
- * IO Utility class for working with Path objects.
+ * Trait that can be mixed in to make an Io utility object, and can be re-used elsewhere.
  */
-object Io {
+trait IoUtil {
   val StdIn   = PathUtil.pathTo("/dev/stdin")
   val StdOut  = PathUtil.pathTo("/dev/stdout")
   val DevNull = PathUtil.pathTo("/dev/null")
 
   /** Creates a new BufferedWriter to write to the supplied path. */
-  def toWriter(path: Path) : BufferedWriter = Files.newBufferedWriter(path)
+  def toWriter(path: Path) : BufferedWriter = new BufferedWriter(new OutputStreamWriter(toOutputStream(path)))
```

Comment:
Yeah - I want all of the `to..()` methods to vector to a single pair of methods that are easy to override and change the behaviour of.  So subclasses/traits can just override `toInputStream()` and `toOutputStream()` and all other methods will pick up those changes.